src/oas.md

```diff
@@ -1157,13 +1176,17 @@ content:
       # Allow an arbitrary JSON object to keep
       # the example simple
       type: object
-    example: {
-      "numbers": [1, 2],
-      "flag": null
-    }
+    examples:
+      minimized:
+        summary: JSON should be serialized with minimal whitespace
+        dataValue: {
+          "numbers": [1, 2],
+          "flag": null
+        }
+        serializedValue: '{"numbers":[1,2],"flag":null}'
```

Comment:
The description of `serializedValue` below says it is "An example of the serialized form of the value, including all relevant encoding and escaping." So why isn't the value shown here the same as what is shown on line 1192?